hazelcast/src/test/resources/log4j2.xml

```diff
@@ -25,11 +25,12 @@
         <Root level="INFO">
             <AppenderRef ref="Console"/>
         </Root>
-        <Logger name="com.hazelcast.instance" level="debug"/>
-        <Logger name="com.hazelcast.cluster" level="debug"/>
-        <Logger name="com.hazelcast.internal.cluster" level="debug"/>
-        <Logger name="com.hazelcast.internal.partition" level="debug"/>
-        <Logger name="com.hazelcast.spi.hotrestart.cluster" level="debug"/>
-        <Logger name="com.hazelcast.test.mocknetwork" level="debug"/>
+        <!-- example how to set log levels -->
```

Comment:
I would either remove the commented section completely or leave here just one commented line as an example.

It would be also fine to mention in a comment the `log4j2-debug.xml` file and also the `ChangeLoggingRule` sample usage.